packages/eslint-plugin/wat.txt

```diff
@@ -0,0 +1,2 @@
+yarn run v1.22.19
+$ /Users/josh/repos/typescript-eslint/node_modules/.bin/jest --json
```

Comment:
should this be delet???

Reply:
wat

Reply:
I swear, the number of times this has happened, I should just check in `wat*` as a `.gitignore` rule to all the repos I work on.